app/views/helpers/pageHeadersAndError.scala.html

```diff
@@ -19,25 +19,26 @@
 @if(backLink.isDefined){
 <a id="backLinkHref" class="link-back" href='@backLink'>@Messages("ated.back")</a>
 @errorDisplay
-<header class="page-header">
-  <h1 id=@headerId class="heading-xlarge">@headerValue</h1>
+    <header class="page-header">
   <p id=@subHeaderId class="heading-secondary"><span class="visuallyhidden">
     @if(subHeaderPrefix.isDefined) {
```

Comment:
I know this wasn't you but would you mind fixing this indentation while you're there please? In fact, this could  just be the following...

@subHeaderPrefix.getOrElse(@Messages("ated.screen-reader.section"))